src/test_utils/katana/genesis.rs

```diff
@@ -291,7 +290,7 @@ impl KatanaGenesisBuilder<Initialized> {
         Ok(self)
     }
 
-    /// Consume the [KatanaGenesisBuilder] and returns the corresponding [GenesisJson].
+    /// Consume the [`KatanaGenesisBuilder`] and returns the corresponding [`GenesisJson`].
```

Comment:
does't this remove the link?

Reply:
Nope, I've tested, it's related to this one https://rust-lang.github.io/rust-clippy/master/index.html#/doc_markdown

And I've another example here https://github.com/paradigmxyz/reth/blob/3eddaf31d0bb13721ca321c36fc074697af0526e/crates/exex/src/manager.rs#L32